Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM cern/cc7-base:20191107
+
+ARG DATE
+ARG VERSION
+
+LABEL org.opencontainers.image.created="${DATE}"
+LABEL org.opencontainers.image.authors='info@reanahub.io'
```

Comment:
We also have `team@reanahub.io` (or `team@reana.io`) which may be nicer to use perhaps.